app/assets/stylesheets/index.scss

```diff
@@ -0,0 +1,35 @@
+ .title {
+  font-size: 20px;
+  margin-bottom: 0px;
+}
+
+.piece {
+  padding-left: 10px;
+  padding-top: 20px;
+  padding-bottom: 0px;
+  margin-left: 10px;
+}
+
+#home {
+  font-size: 40px;
+  padding: 0px;
+}
+
+.piece-title {
+  font-size: 24px;
+  padding: 0px;
```

Comment:
Properties should be ordered font-size, line-height, margin, padding<br>`0px` should be written without units as `0`